otsclient/args.py

```diff
@@ -261,6 +275,10 @@ def parse_ots_args(raw_args):
         pass
 
     args = parser.parse_args(raw_args)
+
+    if bool(args.nonce) ^ bool(args.txid):
+        parser_stamp.error('--nonce and --txid must be given together')
+
```

Comment:
This test doesn't work if you run the `ots` command by itself:

```
$ ots
Traceback (most recent call last):
  File "/home/user/.local/bin/ots", line 33, in <module>
    sys.exit(load_entry_point('opentimestamps-client', 'console_scripts', 'ots')())
  File "/home/user/opentimestamps-client/otsclient/ots.py", line 18, in main
    args = otsclient.args.parse_ots_args(sys.argv[1:])
  File "/home/user/opentimestamps-client/otsclient/args.py", line 279, in parse_ots_args
    if bool(args.nonce) ^ bool(args.txid):
AttributeError: 'Namespace' object has no attribute 'nonce'
```

Reply:
I guess it needs an `if args:` ?

Reply:
I ended up using `hasattr`